### DISCUSSION TITLE
Can i get network tab?

### DISCUSSION THREAD

**Discussion by Fenish** [cefsharp/CefSharp#3482] [Q&A]
Can i get requested urls shows in network tab?
I want to check if spesific url is requested by webpage.

**OS:** Windows 7
**Version:** 88.2.90
**Platform:** Winforms

**Comment by amaitland** [comment]
You can open DevTools to access the network tab see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#javascript-debugging

If you are referring to checking the requests programatically then something like the following can be used

```c#
public class CustomResourceRequestHandler : CefSharp.Handler.ResourceRequestHandler
{
	protected override CefReturnValue OnBeforeResourceLoad(IWebBrowser chromiumWebBrowser, IBrowser browser, IFrame frame, IRequest request, IRequestCallback callback)
	{
		return CefReturnValue.Continue;
	}
}

public class CustomRequestHandler : CefSharp.Handler.RequestHandler
{
	protected override IResourceRequestHandler GetResourceRequestHandler(IWebBrowser chromiumWebBrowser, IBrowser browser, IFrame frame, IRequest request, bool isNavigation, bool isDownload, string requestInitiator, ref bool disableDefaultHandling)
	{
		return new CustomResourceRequestHandler();
	}
}

browser.RequestHandler = new CustomRequestHandler();
```

http://cefsharp.github.io/api/89.0.x/html/T_CefSharp_Handler_ResourceRequestHandler.htm

https://github.com/cefsharp/CefSharp/wiki/General-Usage has lots of examples of the different things you can do with the browser.
